src/inspector_agent.cc

```diff
@@ -243,6 +243,10 @@ class NodeInspectorClient : public v8_inspector::V8InspectorClient {
     return uv_hrtime() * 1.0 / NANOS_PER_MSEC;
   }
 
+  void contextDestroyed(Local<Context> context) {
+    inspector_->contextDestroyed(context);
```

Comment:
Wouldn't the symmetrical thing be to call `inspector_->contextDestroyed(env_->context())` from `~NodeInspectorClient`?